ubuntu_image/common_builder.py

```diff
@@ -111,6 +111,7 @@ def _log_exception(self, name):
             super()._log_exception(name)
 
     def make_temporary_directories(self):
+        _logger.debug(" "*8 + "> DEBUG: workdir: %s", self.workdir)
```

Comment:
Is the whitespace in front of the debugging output really necessary?

Also, we try to use the ' ' quotes instead of " " in string output. It's probably not all consistent, but with new code we try to use this standard notation.

Reply:
i decreased the indentation. And changed the quotes